api/src/config/default.yaml

```
@@ -0,0 +1,3 @@
+server:
+  host: 'localhost'
+  port: 8080
```

HTTP 스펙이 안 나와있는 것 같아서 키를 바꾸면 어떨까요?
```suggestion
http:
  host: 'localhost'
  port: 8080
```

---

https://github.com/MovieReviewComment/Mr.C/pull/14/commits/6addef23cd7ff77b2c0ad4b207fd616d10cf61a4 에서 반영했습니다.